mod-source-record-manager-server/src/main/java/org/folio/services/mappers/processor/functions/NormalizationFunction.java

```diff
@@ -351,6 +353,52 @@ public String apply(RuleExecutionContext context) {
         .map(AlternativeTitleType::getId)
         .orElse(STUB_FIELD_TYPE_ID);
     }
+  },
+
+  SET_ISSUANCE_MODE_ID() {
+
+    @Override
+    public String apply(RuleExecutionContext context) {
+      String subFieldValue = context.getSubFieldValue();
+      char seventhChar = subFieldValue.charAt(6);
+      List<IssuanceMode> issuanceModes = context.getMappingParameters().getIssuanceModes();
+      if (issuanceModes == null || issuanceModes.isEmpty()) {
```

Comment:
```suggestion
      if (CollectionUtils.isEmpty(issuanceModes)) {
```

Reply:
I implement this check like in other Normalization-functions for common approach. But I think that `CollectionUtils.isEmpty()` -  is the best way to do it. So, I will change it in other normalization-function for saving a common approach. Thx @KaterynaSenchenko !

Reply:
I double-checked and found that it was a really specific case. Thank you for this comment!